corehq/apps/linked_domain/tests/test_linked_userreports.py

```diff
@@ -161,4 +167,4 @@ def test_remote_link_ucr(self, fake_ucr_getter):
 
         update_linked_ucr(self.domain_link, linked_report_info.report.get_id)
         report = ReportConfiguration.get(linked_report_info.report.get_id)
-        self.assertEqual("Another new title", report.title)
+        self.assertEqual("CommBugz", report.title)
```

Comment:
I guess this is the correct behavior expected since [this](https://github.com/dimagi/commcare-hq/blob/d68d0f2bc8665f784387979598012658f9b2aece/corehq/apps/linked_domain/ucr.py#L146) but just did what was needed for test to pass.

Reply:
Can you explain this? I think if this isn't showing `Another new title`, then something isn't working here...

Reply:
Ya. I was a bit surprised by this. 
From the code I linked it appears that the title should be the one from master domain, so the update made in the domain's report to set title to "Another new title" will get overwritten.
I didn't look at it further and assumed that the test was not correct earlier since it was not running as expected and wasn't caught.
Can you follow up on this?

Reply:
https://github.com/dimagi/commcare-hq/pull/29654/commits/104d183a3466d0f95dd0fad6e1e5e066c2482a2d